userdocs/diagnostics/multiple-inheritance.md

```diff
@@ -1,6 +1,6 @@
 # Multiple Inheritance
 
-In some programming languages, a class can inherit the interface of multiple base classes. Known as multiple inheritance, this feature can add significant complexity to the language and is unsupported in Swift. Instead, Swift allows composition of interfaces using protocols.
+In some programming languages, a class can inherit the interface of multiple base classes. Known as multiple inheritances, this feature can add significant complexity to the language and is unsupported in Swift. Instead, Swift allows the composition of interfaces using protocols.
```

Comment:
This is not plural, the term of art is `multiple inheritance`

Reply:
`allows composition of interfaces using protocols` is correct. `the composition of interfaces` reads like `composition` is an object.